.github/workflows/check.yml

```diff
@@ -56,7 +57,7 @@ jobs:
         with:
           python-version: "3.11"
       - name: Install tox
-        run: python -m pip install tox pip -U
+        run: python -m pip install --upgrade pip tox
```

Comment:
Revert this orthogonal change.

Reply:
Pip has at least five different options that start with `--u` so I always think it is lazy and not self-documenting to use `-U` unless one it typing on the command line.

Keeping lists sorted makes it easy to spot missing items and nearly impossible insert duplicates.

Reply:
Six...
```
  --user                      Install to the Python user install directory for
  -U, --upgrade               Upgrade all specified packages to the newest
  --upgrade-strategy <upgrade_strategy>
  --use-pep517                Use PEP 517 for building source distributions
  --use-feature <feature>     Enable new functionality, that may be backward
  --use-deprecated <feature>  Enable deprecated functionality, that will be
```

Reply:
Be that so, but is an orthogonal change to this PR.